test/unit/payments/get-payments-by-scheme-id.test.js

```diff
@@ -0,0 +1,26 @@
+jest.mock('../../../app/payments/get-data')
+const { getData: mockGetData } = require('../../../app/payments/get-data')
+
+const { DATA } = require('../../mocks/values/data')
+
+const { SCHEME_ID: SCHEME_ID_CATEGORY } = require('../../../app/constants/categories')
+const { SCHEME_VALUE } = require('../../../app/constants/scheme-value')
+
+const { getPaymentsByScheme } = require('../../../app/payments')
+
+describe('get payments by frn', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    mockGetData.mockResolvedValue(DATA)
+  })
+
+  test('should call mockGetData with schemeId category and value', async () => {
+    await getPaymentsByScheme()
+    expect(mockGetData).toHaveBeenCalledWith(SCHEME_ID_CATEGORY, SCHEME_VALUE)
+  })
+
+  test('should return DATA when getPaymentsBySchemeId is called', async () => {
```

Comment:
Why is data capitalised here? Is It supposed to be referencing the same data as the test name above?

Reply:
I guess it's just not very clear what data is or should be